### ISSUE TITLE
Alt-Tab aka App Switching is broken in Budgie

### ISSUE THREAD

**Issue by rbreaves** [rbreaves/kinto#172]
**Describe the bug**
Broken App Switching in Budgie, initially swaps out the app but then only cycles between 2. The held modifier key is not being detected properly.

**Expected behavior**
Normal App Switching

**Install Type:** VM
**Distro:** Ubuntu Budgie 20.04
**DE:** Budgie
**Branch:** master

**Comment by rbreaves** [comment]
Issue is solved under the Dev branch, however other keybinds such as Ctrl/Cmd/Super + ` or tilde fail to work for in group app switching. Additional patches to Budgie will need to occur.

The current patch can be seen here, and will likely be merged into master before a full patch is created to do the in group app switching as well as that is more of a Budgie limitation than anything else. In group app switching doesn't work right in Budgie even before Kinto remaps the keys.

https://github.com/solus-project/budgie-desktop/compare/master...rbreaves:binaries

Additionally budgie's global menu has an issue where it could get cut off. That too has been patched, but has not made it into Kinto and might not until a better solution is created that can be merged officially into budgie.

https://github.com/solus-project/budgie-desktop/issues/1976

**Comment by rbreaves** [comment]
Closing this ticket, Alt-Tab support was added 12 days ago. https://github.com/rbreaves/kinto/commit/2567f5006e1ec619265d5969748e3978c4ad36a4
